samples/languages/nodejs/nodejs-host_config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "root": {
+        "image_path": "nodejs.img"
+    },
+
+    "tap_device": "sgxlkl_tap0",
```

Comment:
Does this sample require networking? Not really, right?

Reply:
Removed.